.github/workflows/test_build.yml

```diff
@@ -0,0 +1,59 @@
+name: Test Build
+
+on:
+  push:
+
+jobs:
+  test:
+    name: PHP 7.3, Laravel 6.*
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.3
+
+      - name: Cache Composer dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.composer/cache/files
+          key: php-7.3-laravel-6.*-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: php-7.3-laravel-6.*-composer-
+
+      - name: Cache npm dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: npm-
+
+      - name: Copy ENV Laravel Configuration for CI
+        run: php -r "file_exists('.env') || copy('.env.example', '.env');"
+
+      - name: Install dependencies
+        run: |
+          composer require "orchestra/testbench:4.*" --no-interaction --no-update
+          composer update --prefer-dist --no-interaction --no-progress --no-suggest
+          npm install
+
+      - name: Build Assets
```

Comment:
Do we need to build the assets? Or are we just doing this to ensure it doesn't throw any errors?

Reply:
That was my thinking, but there may be better ways to handle this, or it may not be worth doing at this point. Let me know what you think, @mattstauffer

Reply:
If you run the tests without this, and it throws errors, then yah, let's keep this.